bofire/data_models/features/continuous.py

```diff
@@ -20,7 +21,10 @@ class ContinuousInput(NumericalInput):
     type: Literal["ContinuousInput"] = "ContinuousInput"
     order_id: ClassVar[int] = 1
 
-    bounds: Tuple[float, float]
+    bounds: Tuple[float, float] = Field(description="mama")
+    local_relative_bounds: Tuple[
+        Annotated[float, Field(gt=0)], Annotated[float, Field(gt=0)]
+    ] = (math.inf, math.inf)
```

Comment:
I was using `math.inf` because this belongs to the standard python library and to prevent to get serialization problems with pydantic when using a numpy type. No it has to be (`inf`, `inf`) as we are calculating the local bounds by `(max(current_val - local_relative_bounds[0], bounds[0]), min(current_val+local_relative_bounds[1]), bounds[1])`

Reply:
`np.inf` and `math.inf` are both of type `float` and `np.inf == math.inf` is `True`. Serialization should be fine either way. We would just save one import-statement.